src/components/NewRuntimeModal.js

```diff
@@ -453,6 +449,22 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
     return _.find(({ imageType }) => _.includes(imageType, ['Jupyter', 'RStudio']), runtimeDetails?.runtimeImages)?.imageUrl
   }
 
+  getCurrentMountDirectory(currentRuntimeDetails) {
+    if (currentRuntimeDetails?.labels.tool) {
+      return currentRuntimeDetails?.labels.tool === 'RStudio' ? '/home/rstudio' : '/home/jupyter-user/notebooks'
+    } else {
+      return ''
+    }
+  }
+
+  getAboutPDWording(currentRuntimeDetails) {
+    if (this.getCurrentMountDirectory(currentRuntimeDetails) === '') {
```

Comment:
Then rather than checking `this.getCurrentMountDirectory(currentRuntimeDetails) === ''` you can just do `if this.getCurrentMountDirectory(currentRuntimeDetails)`